HACKING.md

```diff
@@ -83,6 +83,39 @@ $ curl --unix-socket ~/pebble/.pebble.socket 'http://localhost/v1/services?names
 ```
 
 
+## Code style
```

Comment:
```snapd``` uses a CODING.md for describing topics related to style. Would it maybe a good idea, as I can see this section growing in the future.

Reply:
Once it grows large enough to justify it, we can break it out into a separate file. No need for premature optimisation IMO.